packages/web/src/pages/userprofile.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { MarketingLayout } from '../components/Layout';
+import queryString from 'query-string';
+import { UserProfileLayout, UserProfileData } from '../components/Layout/UserProfileLayout';
+import { Alert, AlertIcon } from '@chakra-ui/alert';
+
+const UserProfile: NextPage = () => {
+    const [user, setUser] = React.useState<UserProfileData>({name: "", hireable: false, purpose: ""});
+    const [isValidUser, setIsValidUser] = React.useState<boolean>(false); // Flag used to catch whether id entered is in database
+
+    React.useEffect(() => {
+        const fetchStatus = async () => {
+
+            // Assuming that you need a query parameter to see a specific user's profile page (/userprofile?id=#)
+            // Concern?: Goes this route when there are additional parameters after id (/userprofile?id=#&key=value)
```

Comment:
Usually, the API will ignore parameters it doesn't recognize and only use the ones it needs. The `/userprofile` is the root endpoint and everything after the `?` is a query parameter.